gradle/wrapper/gradle-wrapper.properties

```diff
@@ -1,7 +1,5 @@
-#Thu Oct 15 09:18:45 PGT 2020
 distributionBase=GRADLE_USER_HOME
 distributionPath=wrapper/dists
+distributionUrl=https\://services.gradle.org/distributions/gradle-8.2.1-all.zip
 zipStoreBase=GRADLE_USER_HOME
```

Comment:
Sorry I know this is a draft but would it be good to keep the `distributionSha256Sum`

Reply:
Addressed in https://github.com/Automattic/pocket-casts-android/pull/1208/commits/fffb8ab9a83635b4740c227948c1a31e928ce105 by using the release checksum from [this page](https://gradle.org/release-checksums/).